### ISSUE TITLE
Add option to use primary/secondary ui-colors from GTK/Qt theme

### ISSUE THREAD

**Issue by Jacalz** [lupoDharkael/flameshot#505]
I would love if flameahot had an option to select ui-colors from the selected GTK+ or Qt theme that’s installed on the users system. It would make it possible to have a more coherent and nicer looking ui compared to the other parts of the desktop. For this I mean the customizable colors for the screenshot GUI and not the desktop of the configuration utility itself.

**Comment by borgmanJeremy** [comment]
We currently let you theme the colors with the configuration below. Are you saying you would like that automatically set based on the GTK or Qt theme?
![image](https://user-images.githubusercontent.com/46930769/95937219-e69c6300-0d9c-11eb-93b0-d9a264d4921a.png)

**Comment by Jacalz** [comment]
Yeah precisely. It would be great if flameshot could automatically get the correct colors from the current theme :)

**Comment by borgmanJeremy** [comment]
Sounds good to, I hesitate to tag it with "good first issue" because I am not quite sure how to do it robustly across all DE's, but I am open to having this as a feature.
